builder/__tests__/index.test.js

```diff
@@ -0,0 +1,16 @@
+const {sortByLevel} = require("../helpers");
```

Comment:
I'm personally prefer to co-locate test files and source to give more visibility for tests. Will it be possible to move it to `helpers` folder?
If not, then file should be named `helpers.test.js`. Maybe it's a good idea to split helpers by file